### PR TITLE
[CHORE] Add RELEASE title option

### PR DESCRIPTION
**Pull request by attilakiss323**
**Summary**
We need RELEASE if we are creating a PR from alpha to beta or beta to master on mobile app

### CODE REVIEW

**Review by across-the-grid — APPROVED** [review]

File: dangerjs/title.js [hunk 1]
```diff
@@ -14,11 +14,12 @@ const titleError = [
   '- `[DEBUG]` for debugging purposes\n',
   'Automatically:\n',
   '- `[BACK-PROPAGATION]` for important changes which also required in other branches (do not change this title)\n',
+  '- `[RELEASE]` for mobile-app release PRs\n',
```
Comment (across-the-grid) [inline dangerjs/title.js:17]: @attilakiss323 Does this prefix gets set automatically, like the back-propagation, or do we have to set it manually? In that case this should be in the `Secondary` section 🤔

**Comment by capmo-jarvis** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please ensure the package version has been bumped in accordance with [Semantic Versioning](https://semver.org/)
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **This PR has no label(s)**

Please add one (or more) labels to give this PR a better visual context:

- `bug`: for a bug related fix
- `dependencies`: for dependency-updates
- `enhancement`: for new a feature/logic
- `needs_rebase`: to indicate that the PR is out-of-date
- `needs_testing`: to indicate that the PR can be reviewed, but requires a proper QA (manual testing)
- `on_hold`: to indicate that the PR can be reviewed, but the merge is scheduled for sometime later
- `prototype`: to indicate that the PR is there for research/testing, but is not planned to be merged
- `refactoring`: for a code quality improvement
- `security`: to indicate that the PR addresses a security vulnerability
- `setup`: for a repo-/code-/test-related setup
- `wip`: for work-in-progress
- `infra`: for AWS-related changes

img

  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **This PR is not assigned to you and/or someone else**

img

  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 38c0a22d27812f6a2f0aa55dd67a7f5a863897b7
</p>

**Comment by across-the-grid** [comment]
@attilakiss323 lol, did danger.js just complain about itself? What an irony! 😂

**Comment by across-the-grid** [comment]
@attilakiss323 Now it is happy 👍

**Review by miodrage — APPROVED** [review]

**Pull request merged**